queries.sql

```diff
@@ -1,3 +1,25 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+-- Find all animals whose name ends in "mon".
+SELECT * FROM animals WHERE name like '%mon';
+
+-- SELECT name from animals WHERE date_of_birth BETWEEN '2016-01-01' AND '2019-01-01';
+SELECT name from animals WHERE date_of_birth BETWEEN '2016-01-01' AND '2019-01-01';
```

Comment:
- [ ] This query is good. But what if an animal was born on the 21st March 2019? Currently, your query doesn't cover those animals. You only cover animals born on the 1st  day and 1st month of 2016 and 2019.
Please update this query to cover animals born in 2019.

See example:
```sql
SELECT name FROM animals WHERE date_of_birth >= '2016/01/01' AND date_of_birth <= '2019/31/12';
```

Reply:
Hi @mmsesay
This was a good catch! :+1: 
Thank you very much for the feedback. :bow:

Reply:
Good job working on the changes. 
The pleasure is all mine @gtekle.